cmake/templates/pkgConfig.cmake.in

```diff
@@ -95,8 +95,17 @@ if(NOT "@PROJECT_CMAKE_CONFIG_INCLUDE_DIRS@ " STREQUAL " ")
   set(@PROJECT_NAME@_INCLUDE_DIRS "")
   set(_include_dirs "@PROJECT_CMAKE_CONFIG_INCLUDE_DIRS@")
   foreach(idir ${_include_dirs})
-    if(IS_ABSOLUTE ${idir} AND IS_DIRECTORY ${idir})
-      set(include ${idir})
+    if(IS_ABSOLUTE ${idir})
+      get_filename_component(path ${idir} DIRECTORY)
+      get_filename_component(idir ${idir} NAME)
+      set(include "include-NOTFOUND")
+      find_path(include ${idir}
+        PATHS ${path}
+        NO_DEFAULT_PATH)
```

Comment:
You are using `find_path` here to check if the absolute path is available under any of the root paths, right?